src/main/java/org/acra/ACRAConfiguration.java

```diff
@@ -320,6 +321,12 @@ public ACRAConfiguration setResDialogNegativeButtonText(int resId) {
         return this;
     }
 
+    @SuppressWarnings( "unused" )
+    public ACRAConfiguration setReportDialogClass(Class<? extends CrashReportDialog> reportDialogClass) {

```

Comment:
For instance what are you planning to do at
  // do something
in https://gist.github.com/timrae/227aa5240f03cf07035d


ᐧ

On Fri, Jan 16, 2015 at 7:30 PM, npombourcq <notifications@github.com>
wrote:

> In src/main/java/org/acra/ACRAConfiguration.java
> <https://github.com/ACRA/acra/pull/218#discussion-diff-23070103>:
>
> > @@ -320,6 +321,12 @@ public ACRAConfiguration setResDialogNegativeButtonText(int resId) {
> >          return this;
> >      }
> >
> > +    @SuppressWarnings( "unused" )
> > +    public ACRAConfiguration setReportDialogClass(Class<? extends CrashReportDialog> reportDialogClass) {
>
> Is there any specific reason to force a subclass of CrashReportDialog?
> Otherwise specifying it as Class<? extends Activity> would make it a lot
> more flexible as one would be able to provide a completely different
> Activity.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/ACRA/acra/pull/218/files#r23070103>.
>